wgsl/index.bs

```diff
@@ -8027,76 +8238,76 @@ value with the same sign.
 <table class='data'>
   <caption>Accuracy of expressions</caption>
   <thead>
-    <tr><th>Expression<th>Accuracy<th>
+    <tr><th>Expression<th>Accuracy for f32<th>Accuracy for f16
   </thead>
 
-  <tr><td>`x + y`<td>Correctly rounded
```

Comment:
Nit: the layout of these tables is a bit hard to read. For the rows where the accuracy of the f16 and f32 are described in the same way (e.g. colspan=2) could we left align the accuracy to make the tables easier to read?

Reply:
Done.